### ISSUE TITLE
shift to up/down workspace options

### ISSUE THREAD

**Issue by jonorthwash** [hernejj/workspace-grid-cinnamon-applet#19]
As of cinnamon 3.0 (I think), options in control center for "shift to up/down workspace" appear to not be included.  The applet's shortcuts seem to still work, but there's now no way to change them as the "navigating" section of the docs suggestion.

I also filed a bug with [cinnamon-control-center](https://github.com/linuxmint/cinnamon-control-center/issues/144), but the devs in #linuxmint-devs didn't seem keen on keeping something like this for a single plugin.

Also note the associated problem with panning animations: instead of up/down animations, it does up+left, down+right animations.

**Comment by hernejj** [comment]
Thanks for the report. I have not tested on Cinnamon 3.0 yet. It looks like I might have some work to do :)

**Comment by RafalSkolasinski** [comment]
Yes, up/down animations are really annoying now. Will there be any update soon? :)

**Comment by hernejj** [comment]
I was going to say "what? I don't see any problem with animation".... then I turned on "Window Effects" in the Effects settings window. Now I see what you guys mean :)

As a short term solution, consider turning off the Window Effects. I'll see if there is anything I can do to fix this.

**Comment by Heidistein** [comment]
Hmm, It seems that I have lost the entire shortcut for up/down (left/right works as expected).
Is there a way, perhapd via dbus, to set the shortcut? Via the preferences of the workspace-switcher applet seems not to work.

**Comment by Lamieur** [comment]
@Heidistein 

> It seems that I have lost the entire shortcut for up/down (left/right works as expected).

I've provided a quick patch in issue #22 - hopefully @hernejj notices, but in the meantime you can just patch it yourself in your .local/share/cinnamon/applets/workspace-grid@hernejj
